dev/com.ibm.ws.microprofile.config.1.2_fat_tck/publish/tckRunner/tck/pom.xml

```diff
@@ -60,6 +68,14 @@
             <systemPath>${com.ibm.websphere.org.eclipse.microprofile.config.1.2.1_}</systemPath>
         </dependency>
 
+       <dependency>
+            <groupId>com.ibm.ws</groupId>
+            <artifactId>fattest.simplicity</artifactId>
+            <version>1</version>
+            <scope>system</scope>
+            <systemPath>${project.basedir}/../../../lib/fattest.simplicity.jar</systemPath> <!-- This will point to build/libs/autoFVT/lib/fattest.simplicity.jar -->
+        </dependency>
+
```

Comment:
It looks like we've included this block twice (just in this pom)

Reply:
This hasn't been addressed. It's less important as it's just a duplicate dependency, but if you were re-running the build anyway it would be good to remove it.